content/soarian/r4/clinical/medications/medicationrequest.md

```diff
@@ -0,0 +1,132 @@
+---
+title: MedicationRequest | R4 API
+---
+
+# MedicationRequest
+
+* TOC
+{:toc}
+
+## Overview
+
+The MedicationRequest resource provides discharge medication orders and administration instructions for a patient in the context of a visit. 
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query.
+
+The search results include the following fields if they contain values:
+
+* [MedicationRequest id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.status){:target="_blank"}
+* [Intent](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.intent){:target="_blank"}
+  * Details in Implementation Notes sections for reads and searches.
+* [Identifier](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.identifier){:target="_blank"}
+* [Reported Boolean](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.reported_x_){:target="_blank"}
+* [Medication](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.medication_x_){:target="_blank"}  (contained)
+  * [Medication code](https://hl7.org/fhir/R4/medication-definitions.html#Medication.code){:target="_blank"}
+* [Patient](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.subject){:target="_blank"}
+* [Encounter](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.encounter){:target="_blank"}
+* [Requester](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.requester){:target="_blank"}
+* [Dosage Instructions](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dosageInstruction){:target="_blank"}
+  * [Text](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.text){:target="_blank"}
+
+## Terminology Bindings
+
+#### Contained Medication Bindings
+
+<%= terminology_table(:soarian_medication_request, :r4) %>
+
+## Search
+
+Search for MedicationRequest resources that meet the specified query parameters
```

Comment:
Please add a period (.) to the end of this sentence.

PRODOC-160857.

Reply:
Done